src/components/SearchForm.vue

```diff
@@ -7,12 +7,14 @@
       placeholder="Type a dog breed..."
       ref="searchInput"
       @input="debounceSearch"
+      :disabled="isDisabled"
```

Comment:
Dodal sem prop kot attribute v search inputu in potem sem ta podatek uporabil v breedslistu, `isInputDisabled`. Če ne dobim podatkov iz fetchBreeds se ta `isInputDisabled` spremeni v `true`, initially is false